src/components/vuestic-components/va-infinite-scroll/VaInfiniteScroll.vue

```diff
@@ -0,0 +1,226 @@
+<template>
+  <component
+    :is="tag"
+    class="va-infinite-scroll"
+    :class="{'va-infinite-scroll--reversed': reverse}"
+  >
+    <slot
+      name="default"
+      ref="content"
+    />
+    <slot
+      name="loading"
+      v-if="!disabled"
+    >
+      <div ref="defaultSpinner"
+           class="va-infinite-scroll__spinner"
+           :class="{'va-infinite-scroll__spinner--invisible': !fetching}">
+        <va-progress-circle
+          size="small"
+          :thickness="0.15"
+          :color="error ? $themes.danger : $themes.primary"
+          indeterminate
+        />
+      </div>
+    </slot>
+  </component>
+</template>
+
+<script>
+import * as _ from 'lodash'
+import VaProgressCircle from '../va-progress-bar/progress-types/VaProgressCircle'
+import { makeContextablePropsMixin } from '../../context-test/context-provide/ContextPlugin'
+import { sleep } from '../../../services/utils'
+
+export default {
+  name: 'VaInfiniteScroll',
+  components: { VaProgressCircle },
+  mixins: [
+    makeContextablePropsMixin({
+      offset: {
+        type: Number,
+        default: 1,
+      },
+      reverse: {
+        type: Boolean,
+        default: false,
+      },
+      disabled: {
+        type: Boolean,
+        default: false,
+      },
+      scrollTarget: {
+        type: [Element, String],
+        default: null,
+      },
+      debounce: {
+        type: Number,
+        default: 100,
+      },
+      load: {
+        type: Function,
+        default: () => {},
+      },
+      tag: {
+        type: String,
+        default: 'div',
+      },
+    }),
+  ],
+  data () {
+    return {
+      index: 0,
+      fetching: false,
+      error: false,
+      initialHeight: null,
+    }
+  },
+  watch: {
+    disabled (value) {
```

Comment:
That seems to be broken. 
![image](https://user-images.githubusercontent.com/5394573/78075919-11e5d480-73ae-11ea-9cf5-f57c600cac9f.png)